blog/2025-05-27-may-releases.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: May 2025 Releases
+slug: 2025-05-27-may-releases
+authors: [travishathaway]
+tags: [announcement, conda, conda-build, conda-libmamba-solver, conda-index]
+description: |
+  conda 25.5.0, conda-build 25.5.0, conda-libmamba-solver 25.4.0 and conda-index 0.6.1 have been released! 🎉
+image: img/blog/2025-05-27-may-releases/banner.png
+---
+
+The May 2025 releases included updates to conda, conda-build, conda-libmamba-solver, and conda-index! 🎉 All of these have been released to both `main` and `conda-forge` channels.
+
+<!-- truncate -->
+
+## Changes in conda [25.5.0](https://github.com/conda/conda/releases/tag/25.5.0)
+
+To update `conda` to the latest version, run:
+
+```bash
+conda install -n base conda=25.5.0
+```
+
+**Notable Changes:**
+- Add `environment_specifiers` plugin hooks to allow plugin authors to read different file formats and sources.
+- Add a plugin hook for `PrefixData` loaders and use it to wrap the existing PyPI interoperability features. These are now present in `conda.plugins.prefix_data_loaders.pypi`.
+- Add a `--fields` flag to conda list that allows to customize the content of the `human` output format (i.e. the default output format).
```

Comment:
```suggestion
- Add a `--fields` flag to conda list that allows to customize the content of the `human` output format (i.e., the default output format).
```
Double punctuation looks weird, but is standard for these latin abbreviations, since they tend to mean  "for example," or "that is,".